pkgs/development/python-modules/pyfcm/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, requests-toolbelt
+}:
+
+buildPythonPackage rec {
+  pname = "pyfcm";
+  version = "1.4.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "cce6c999d8ffe39a44c42329f40c73306586102c6379fc726ea217070b72b967";
+  };
+
+  propagatedBuildInputs = [ requests-toolbelt ];
+
+  # Pypi does not provide test files
+  doCheck = false;
```

Comment:
If they do releases on github, i would pull from there
```
src = fetchFromGitHub {
  owner = "olucurious";
  repo = "pyfcm";
  rev = "v${version}";
  sha256 = "...";
```

Reply:
That's fair. I'll get around changing this as soon as I'm free. Thanks for the review!

Reply:
@jonringer Testing implies communication with an external API (Google's FCM), using a valid API key. Is this basis to disable testing altogether or should I look for a way to disable those tests?

Reply:
usually there's a subsets that has unittest. This helps validate that certain things like import paths haven't changed.

Reply:
However, I just looked at their code, and it looks like it's solely centered around pinging the API. I would change your comment to also say that the unittests also require network access.

Reply:
There we go. Made it a GitHub build just in case the test suite is modified later on, and we can make use of it.